src/Products/MeetingLalouviere/browser/overrides.py

```diff
@@ -306,6 +306,57 @@ def get_categories_for_commission(self, commission_num):
             # single category as a string
             return [cat]
 
+    def get_commission_items(self, itemUids, commission_num, type='normal'):
+        """
+        Get the items of the commission
+        :param commission_num: number of the commission
+        :param supplement: supplement items
+        :param type: must be 'normal', 'supplement' or '*'
+        :return: list of meetingItem
+        """
+        cats = self.get_categories_for_commission(commission_num)
+        print cats
+        if type == 'supplement':  # If we want the supplements items only
+            cats = [cat + '-1er-supplement' for cat in cats]  # append supplement suffix to the categories
+        elif type == '*':  # If we want all items
+            cats = [cat + '-1er-supplement' for cat in cats] + cats
```

Comment:
Order should be usual categories first. Then all supplements in the exact same order than the usual

Reply:
Yep, done 
(hopefully, it's good :sweat_smile: )